src/features/mcda/parser/adapters.ts

```diff
@@ -1,11 +1,14 @@
 import type { MCDAConfig } from '../types';
-import type { JsonMCDAv1, JsonMCDAv2, JsonMCDAv3 } from './types';
+import type { JsonMCDAv1, JsonMCDAv2, JsonMCDAv3, JsonMCDAv4 } from './types';
 
-export function firstVersionAdapter(json: JsonMCDAv1): MCDAConfig {
+export function firstVersionMigration(json: JsonMCDAv1): MCDAConfig {
```

Comment:
will naming like JsonMCDAv1Migration or MCDAv1Migration look more understandable?

Reply:
because these functions essentially describe the migration of older versions to newer ones